contracts/DispenserProvider.sol

```diff
@@ -14,6 +14,8 @@ contract DispenserProvider is DispenserModifiers {
     }
 
     /// @notice Dispenses tokens from a locked pool based on provided data and signature.
+    /// If the pool owner wants to dispense tokens from the pool, it is better to use the Withdraw option
+    /// or Split and then Withdraw option.
```

Comment:
I think the point is for "himself"
the owner still can use this to dispense to someone else

and the signature is the same - and only can be used 1 time.